en/docs/concepts/authentication/multi-factor-authentication.md

```diff
@@ -0,0 +1,46 @@
+# Multi-Factor Authentication
+
+## What is multi-factor authentication?
+
+Multi-factor authentication (MFA) is an effective method of providing enhanced security during user authentication by verifying user identity using more than one piece of identifying information. This means that in addition to providing basic authentication (username and password credentials), the user is also prompted to provide further verfication in the form of a one-time password, biometrics, hardware key devices etc. 
+
+Due to increasing digital crimes and internet fraud in the world, people are highly concerned about security. The traditional username and password method is not enough to secure the authentication. Processing speeds of CPUs have increased, so brute force attacks are a reality and dictionary attacks have become a common threat. GPGPU password cracking and rainbow tables have provided similar advantages to attackers.
+
+Thereby, MFA creates a layered defense and makes it more difficult for an unauthorized person to gain access. Even if the username and password is compromised, the attacker would still have atleast one more barrier to break through in order to successfully breach security. 
+
+---
+
+## Authetication factors
+
+Multi-factor authentication uses a combination of different types of authentication factors. The three main categories are: 
+
+- Knowledge factors: Things only the user knows, such as passwords or PIN numbers.
+
+- Possession factors: Things only the user has, such as ATM cards or hardware key devices.
+
+- Inherence factors: Things only the user is, such as a fingerprint or face.
+
+A common day-to-day example of using a combination of these factors for MFA can be seen when withdrawing money from a bank. In this case, the ATM card is the possession factor and the PIN number is the knowledge factor.
+
+---
+
+## Using multi-factor authentication
+
+By default, WSO2 Identity Server (WSO2 IS) is shipped with username-and-password-based authentication along with support for adding additional steps of authentication with authenticators such as SMSOTP, FIDO, MEPin, etc. WSO2 Identity Server allows configuring multi-step authentication that enables defining an authentication chain containing different authenticators in different steps. This chain can also be converted to a multi-factor authentication chain by adding different factors of authentication as options to each step. 
```

Comment:
Shall we give an example to differentiate multi-step authentication and multi-factor authentication here?

Reply:
any ideas for an example? the main difference is just that the authenticators can either be multiple steps that have to be followed or multiple options within a step

Reply:
I felt that the user might be confused with "steps" and "factors" since this is just a basic document. 

If we could just give an example where the same factor is used in different steps (say pin and password) and then say this is not called a multi-factor authentication unless one of the factors here changes (Replace pin with fingerprint).